ichnaea/locate/provider.py

```diff
@@ -553,6 +554,8 @@ class FallbackProvider(Provider):
     log_name = 'fallback'
     location_type = Position
     source = DataSource.Fallback
+    RATE_LIMIT_EXPIRE = 60
+    CACHE_EXPIRE = (24 * 60 * 60)  # 1 day
```

Comment:
I'd prefer to get the cache expiry time from the config file. It should be possible to set the expiry to either None or 0 to disable this sort of caching. Going forward we want to make ichnaea more useful to third parties running the code, and that includes the little details.